From: Jes Sorensen <Jes.Sorensen@redhat.com>
Date: Thu, 26 Jun 2014 10:20:53 +0200
Subject: [174/249] staging: rtl8723au: sreset_priv->dbg_trigger_point is never
 set
Origin: https://git.kernel.org/linus/e899c9176a1d0659d854802966f1b1a8e74394a6

No point carrying a bunch of code around that is never going to get
called.

Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723au/core/rtw_sreset.c     |  7 -------
 drivers/staging/rtl8723au/hal/rtl8723a_sreset.c | 14 --------------
 drivers/staging/rtl8723au/include/rtw_sreset.h  |  8 --------
 3 files changed, 29 deletions(-)

diff --git a/drivers/staging/rtl8723au/core/rtw_sreset.c b/drivers/staging/rtl8723au/core/rtw_sreset.c
index c19831c..c152aef 100644
--- a/drivers/staging/rtl8723au/core/rtw_sreset.c
+++ b/drivers/staging/rtl8723au/core/rtw_sreset.c
@@ -37,13 +37,6 @@ void rtw_sreset_reset_value(struct rtw_adapter *padapter)
 	psrtpriv->last_tx_complete_time = 0;
 }
 
-void sreset_set_trigger_point(struct rtw_adapter *padapter, s32 tgp)
-{
-	struct hal_data_8723a	*pHalData = GET_HAL_DATA(padapter);
-
-	pHalData->srestpriv.dbg_trigger_point = tgp;
-}
-
 bool rtw_sreset_inprogress(struct rtw_adapter *padapter)
 {
 	struct rtw_adapter *primary_adapter = GET_PRIMARY_ADAPTER(padapter);
diff --git a/drivers/staging/rtl8723au/hal/rtl8723a_sreset.c b/drivers/staging/rtl8723au/hal/rtl8723a_sreset.c
index 4374036..54a08a1 100644
--- a/drivers/staging/rtl8723au/hal/rtl8723a_sreset.c
+++ b/drivers/staging/rtl8723au/hal/rtl8723a_sreset.c
@@ -52,22 +52,8 @@ void rtl8723a_sreset_xmit_status_check(struct rtw_adapter *padapter)
 			}
 		}
 	}
-
-	if (psrtpriv->dbg_trigger_point == SRESET_TGP_XMIT_STATUS) {
-		psrtpriv->dbg_trigger_point = SRESET_TGP_NULL;
-		rtw_sreset_reset(padapter);
-		return;
-	}
 }
 
 void rtl8723a_sreset_linked_status_check(struct rtw_adapter *padapter)
 {
-	struct hal_data_8723a	*pHalData = GET_HAL_DATA(padapter);
-	struct sreset_priv *psrtpriv = &pHalData->srestpriv;
-
-	if (psrtpriv->dbg_trigger_point == SRESET_TGP_LINK_STATUS) {
-		psrtpriv->dbg_trigger_point = SRESET_TGP_NULL;
-		rtw_sreset_reset(padapter);
-		return;
-	}
 }
diff --git a/drivers/staging/rtl8723au/include/rtw_sreset.h b/drivers/staging/rtl8723au/include/rtw_sreset.h
index 4799614..60fa829 100644
--- a/drivers/staging/rtl8723au/include/rtw_sreset.h
+++ b/drivers/staging/rtl8723au/include/rtw_sreset.h
@@ -18,19 +18,11 @@
 #include <osdep_service.h>
 #include <drv_types.h>
 
-enum {
-	SRESET_TGP_NULL = 0,
-	SRESET_TGP_XMIT_STATUS = 1,
-	SRESET_TGP_LINK_STATUS = 2,
-};
-
 struct sreset_priv {
 	struct mutex	silentreset_mutex;
 	u8	silent_reset_inprogress;
 	unsigned long last_tx_time;
 	unsigned long last_tx_complete_time;
-
-	s32 dbg_trigger_point;
 };
 
 #include <rtl8723a_hal.h>
